Xero.NetStandard.OAuth2/Model/PayrollUk/Employment.cs

```diff
@@ -123,6 +123,13 @@ public enum NiCategoryEnum
         /// <value>The NI Category of the employee</value>
         [DataMember(Name="niCategory", EmitDefaultValue=false)]
         public NiCategoryEnum NiCategory { get; set; }
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Employment" /> class.
+        /// </summary>
+        [JsonConstructorAttribute]
+        public Employment() 
```

Comment:
why are these classes generating now. We just fixed tests, let's discuss.

Reply:
These classes are generated from Open API, not from the code-gen. we've corrected some required fields and after that change these classes has generated